wolfssl/wolfcrypt/ed448.h

```diff
@@ -86,6 +89,12 @@ struct ed448_key {
 #ifdef WOLFSSL_ASYNC_CRYPT
     WC_ASYNC_DEV asyncDev;
 #endif
+#if defined(WOLF_CRYPTO_CB)
+    int devId;
+#endif
+    void *heap;
+    wc_Shake sha;
+    int sha_clean_flag;
```

Comment:
This could technically be `byte` or even`byte sha_clean_flag:1`.

Reply:
as discussed, leaving this `int` since the compiler would likely do this anyway to keep the struct size aligned, and leaving it `int` avoids runtime byte/bit masking/shuffling.